lib/geoblacklight/item_viewer.rb

```diff
@@ -14,41 +14,43 @@ def viewer_endpoint
       viewer_preference.values.first.to_s
     end
 
-    def wms
-      @references.wms
-    end
+    private
 
-    def iiif
-      @references.iiif
-    end
+      def wms
```

Comment:
Is this a rubocop thing? I'd be fine to ignore if it makes things simpler for you.

Reply:
Not a rubocop cop thing, just a personal preference/reflex. We tend to do this in Princeton projects as well. I can adjust this to match the rest of the codebase.

Reply:
Gotcha. I'd be fine for this maybe in another PR.. but adding the style things here makes the review a bit more difficult.

Reply:
Actually, rubocop is complaining about this. The `Layout/IndentationWidth` cop which seems a bit general to turn off for the whole file. We have to use `Layout/IndentationConsistency` and set `EnforcedStyle: indented_internal_methods`

Reply:
Or not... yeah, we'll just have to exclude the file from the `IndentationConsistency` cop.

Reply:
Fixed.